convert2rhel/actions/post_conversion/hostmetering.py

```diff
@@ -0,0 +1,238 @@
+                " Please note that this option is mainly used for testing and will configure host-metering unconditionally. "
+                " For generic usage please use the 'auto' option."
+            )
+            self.add_message(
+                level="WARNING",
+                id="FORCED_CONFIGURE_HOST_METERING",
+                title="The `force' option has been used for the CONVERT2RHEL_CONFIGURE_HOST_METERING environment variable.",
+                description="Please note that this option is mainly used for testing and"
+                " will configure host-metering unconditionally."
+                " For generic usage please use the 'auto' option.",
+            )
+        elif self.env_var == "auto":
+            logger.debug("Automatic detection of host hyperscaler and configuration.")
+
+        return True
+
+    def is_running_on_hyperscaler(self, rhsm_facts):
+        """
+        Check if the system is running on hyperscaler. Currently supported
+        hyperscalers are aws, azure and gcp.
+
+        :param rhsm_facts: Facts about the system from RHSM.
+        :type rhsm_facts: dict
+        :return: True if the system is running on hyperscaler, False otherwise.
+        :rtype: bool
+        """
+        is_aws = rhsm_facts.get("aws_instance_id")
+        is_azure = rhsm_facts.get("azure_instance_id")
+        is_gcp = rhsm_facts.get("gcp_instance_id")
+        return any([is_aws, is_azure, is_gcp])
+
+    def _enable_host_metering_service(self):
+        """
+        Enables and starts the host metering service.
+        Return command and it's output in case of fail.
+
+        Example:
+            Command called: systemctl enable host-metering.service
+            Return values:
+                (command, message)
+                command = "systemctl enable host-metering.service"
+                message = "Some message with error description."
+
+        :return: Empty string if host-metering is enabled and started successfully. Otherwise failed command and it's output if available.
+        :rtype: tuple(str, str)
+        """
+
+        logger.info("Enabling host-metering service.")
+        command = ["systemctl", "enable", "host-metering.service"]
+        output, ret_enable = run_subprocess(command)
+        if output:
+            logger.debug("Output of systemctl call: %s" % output)
+        if ret_enable:
+            logger.warning("Failed to enable host-metering service.")
+            return " ".join(command), output
+
+        logger.info("Starting host-metering service.")
+        command = ["systemctl", "start", "host-metering.service"]
+        output, ret_start = run_subprocess(command)
+        if output:
+            logger.debug("Output of systemctl call: %s" % output)
+        if ret_start:
+            logger.warning("Failed to start host-metering service.")
+            return " ".join(command), output
+
+        # Any command failed, no error output
```

Comment:
```suggestion
        # All commands succeeded, no error output found
```

Reply:
Yeah, that sound's better :D I removed the negation there https://github.com/oamg/convert2rhel/pull/1311#discussion_r1706921105 but added here... :D